ui-modules/utils/server-status/server-status.js

```diff
@@ -33,13 +33,15 @@ angular.module(MODULE_NAME, [uibModal])
 
 export default MODULE_NAME;
 
+const LOGIN_PAGE_HEADER = "X_BROOKLYN_LOGIN_PAGE";
```

Comment:
@duncangrant Is that always returned from the API? I'm asking because it is used to build up the link to redirect to

Reply:
@duncangrant So if the header is not set, it will redirect to the home page. Is that the expected behaviour?